README.md

```diff
@@ -0,0 +1,23 @@
+# The Ipe extensible drawing editor
+
+[![Flatpak](https://github.com/flathub/org.otfried.Ipe/actions/workflows/flatpak.yml/badge.svg)](https://github.com/flathub/org.otfried.Ipe/actions/workflows/flatpak.yml)
+[![Check for updates](https://github.com/flathub/org.otfried.Ipe/actions/workflows/update.yml/badge.svg)](https://github.com/flathub/org.otfried.Ipe/actions/workflows/update.yml)
+
+[Download latest](https://nightly.link/flathub/org.otfried.Ipe/workflows/flatpak/master/ipe-x86_64.zip)
+
+Official website: https://ipe.otfried.org/
+
+Ipe is a drawing editor for creating figures in PDF format. It supports making
+small figures for inclusion into LaTeX-documents as well as making multi-page
+PDF presentations.
+
+## Using TeX Live
+
+There is no TeX Live installation bundled with this Flatpak, because it is
+assumed that most users have a Tex Live installation already.  Instead here is
+how to forward an existing TeX Live installation into the sandbox and making Ipe
+aware of where to find it.
```

Comment:
there is a SDK for that reason. What you are recommending here is to punch a hole in the sandbox.

BTW no user will see this readme.

Reply:
How is this punching a hole in the sandbox? The bind-mounted binaries are going to be run inside the sandbox.

Anyway, I will try to get this working with the TeX Live SDK.

Reply:
any bind mount like that is a hole in the sandbox. 

Also the "requirement" comes against what flatpak provides.

Reply:
I respectfully disagree. There is no difference between bind-mounting host binaries into the sandbox and executing them there and bind-mounting an SDK into the sandbox. Nevertheless, I adjusted the README and the SDK solution is now promoted.